_posts/newsletters/2020-04-13-newsletter.md

```diff
@@ -0,0 +1,77 @@
+
+The sessions will be held using Zoom. Connection information will be sent to registered participants before the workshop. (We've seen the recent news about Zoom security issues, and we will be taking appropriate security precautions.)
+
+If you have questions or would like to contribute a talk to one of the sessions, please contact [contact@us-rse.org](mailto:contact@us-rse.org) or reach out via slack. 
+
+<a name="princeton"></a>
+# US-RSE Association Workshop at Princeton is moving to October
+
+The in-person [US-RSE Community Building Workshop](https://us-rse.org/first-community-workshop/) has been officially rescheduled for October 27-28, 2020. It will be held at the same venue, the Nassau Inn in Princeton, NJ.
+
+This is a rapidly changing situation and it is impossible to predict what the travel/conference landscape will look like in October. Should COVID-19 continue to prevent us from having the workshop in October, we may need to reschedule for sometime in the following 12 months. Though we certainly hope it doesn't come to that.
+
+Follow the [US-RSE Event Page](https://us-rse.org/events-training/) to stay up to date.
+
+<a name="workshops"></a>
+# Workshops at PEARC20 and SC20
+
+We are thrilled to announce that in addition to the [PEARC20](https://us-rse.org/events/2020/2020-07-pearc20) workshop, we will have a workshop at [SC20](https://sc20.supercomputing.org/)! The PEARC20 workshop will be a Research Software Engineers Community Workshop, and the SC20 workshop will be "Research Software Engineers in HPC: Creating Community, Building Careers, Addressing Challenges (RSE-HPC-2020)." Stay tuned for more information.
+
+<a name="jobs"></a>
+# Recent Job Postings
+
+These opportunities were recently posted to the [RSE Careers page](https://us-rse.org/jobs/):
+
+*   [Architect, PACE team at Georgia Institute of Technology](https://pace.gatech.edu/xsedeosg-architect); Atlanta, GA
+*   [Scheduler Architect, PACE team at Georgia Institute of Technology](https://pace.gatech.edu/scheduler-architect); Atlanta, GA
+*   [Staff Scientist/Software Engineer at New Mexico Tech](https://www.nmt.edu/hr/docs/hr/jobs/StaffSciSoftEngIRIS20-030.pdf); Socorro, NM
+*   [Senior Research Software Engineer (Data Engineer)](http://bit.ly/fasrc_senior_rse); Cambridge, MA
+*   [Data Scholars](https://datascience.nih.gov/data-scholars); NIH, Bethesda and Rockville, MD, or Research Triangle Park, NC
+
+<a name="reads"></a>
+# Interesting Reads
+
+Items you may have missed on the blog and Slack:
+
+These papers might be of interest to RSEs: 
+
+*   J. Cohen, D. S. Katz, M. Barker, N. Chue Hong, R. Haines, C. Jay, "The Four Pillars of Research Software Engineering," IEEE Software, [https://doi.org/10.1109/MS.2020.2973362](https://doi.org/10.1109/MS.2020.2973362) (preprint: [https://arxiv.org/abs/2002.01035](https://arxiv.org/abs/2002.01035))
```

Comment:
I haven't looked yet, but each citation might render more nicely with a code block (one per citation).

Reply:
Yep I agree with this sentiment after viewing the list - these bullets would be very clean (and akin to how you might see in a box to copy paste them on some site) rendered in separate code blocks:

![image](https://user-images.githubusercontent.com/814322/79159096-b85dac80-7d94-11ea-90cc-58ac477ed27b.png)

Reply:
The other option is to just list the papers (by Lastname .etal) instead of putting the entire citation. Is the citation really needed in this context? Could the reader not get the official citation by clicking the link?